libs/eo/help/shell/src/lib/eo-geography-page.component.ts

```diff
@@ -15,12 +15,192 @@
  * limitations under the License.
  */
 import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { EoStackComponent } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
+import { eoRoutes } from '@energinet-datahub/eo/shared/utilities';
 
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
+  imports: [EoStackComponent, RouterModule],
   selector: 'eo-geography-page',
-  styles: [``],
-  template: ` <div>Geography</div> `,
+  styles: [
+    `
+      :host {
+        display: grid;
+        grid-auto-flow: column;
+        gap: var(--watt-space-l);
+        grid-template-columns: 600px 360px; // Magic UX Numbers
+      }
+
+      .column {
+        display: flex;
+        flex-direction: column;
+        gap: var(--watt-space-l);
+      }
+
+      .img-box {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        font-size: 14px;
+        gap: var(--watt-space-s);
+      }
+
+      .text-box {
+        padding: var(--watt-space-m);
+      }
+
+      .case {
+        display: flex;
+        background-color: var(--watt-color-primary-light);
+
+        p {
+          padding: var(--watt-space-m);
+
+          b {
+            color: var(--watt-color-primary-dark);
+          }
+        }
+      }
+
+      .info-box {
+        padding: var(--watt-space-m);
+
+        &.green {
+          background-color: var(--watt-color-primary-dark);
+          color: white;
+
+          span {
+            color: white;
+          }
+        }
+
+        &.beige {
+          background-color: var(--watt-color-secondary-light);
+        }
+
+        img {
+          display: block;
+          padding-bottom: var(--watt-space-m);
+        }
+      }
+    `,
+  ],
+  template: `
+    <div class="column">
+      <div class="text-box">
+        <eo-stack size="M">
+          <div>
+            <span class="watt-headline-3"
```

Comment:
This should be a header.
h1, if none exist already, otherwise h2 etc.
Same applies to the rest of the headlines throughout the templates.

Reply:
Fixed :)